tests/components/ring/device_mocks.py

```diff
@@ -158,6 +158,9 @@ def update_history_data(fixture):
         mock_device.configure_mock(
             siren=device_dict["siren_status"].get("seconds_remaining")
         )
+        mock_device.async_set_siren.side_effect = lambda i: mock_device.configure_mock(
+            siren=i
+        )
```

Comment:
This is required because of the addition of the `refresh_after` step which brings the api state inline with the hass state.